rust/src/lib/ieee8021x.rs

```diff
@@ -51,3 +53,30 @@ impl Ieee8021XConfig {
         }
     }
 }
+
+impl std::fmt::Debug for Ieee8021XConfig {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let mut cloned = self.clone();
+        cloned.hide_secrets();
+        let debug_tr = format!(
+            "
+                Ieee8021XConfig {{
+                    identity: {:?},
+                    eap: {:?},
+                    private_key: {:?},
+                    client_cert: {:?},
+                    ca_cert: {:?},
+                    private_key_password: {:?}
+                }}
+            ",
+            cloned.identity,
+            cloned.eap,
+            cloned.private_key,
+            cloned.client_cert,
+            cloned.ca_cert,
+            cloned.private_key_password
+        );
+
+        write!(f, "{}", debug_tr)
```

Comment:
In https://doc.rust-lang.org/std/fmt/trait.Debug.html , `f.debug_struct` is suggested.

Reply:
Wow, Thanks